src/main.rs

```diff
@@ -59,7 +98,10 @@ pub fn main() {
                 Event::Quit {..} | Event::KeyDown { keycode: KeyCode::Escape, .. } => {
                     running = false
                 },
-                Event::Window { win_event_id: WindowEventId::Resized, .. } => { was_resized = true },
+                Event::Window { win_event_id: WindowEventId::Resized, data2, .. } => {
+                    window_height = data2;
```

Comment:
Is ```data1``` the width?  If so, we can just get that as well and skip the whole ```win_properties``` thing below since that's all we get from that is the new width and height.